release-notes/5.0/5.0-known-issues.md

```diff
@@ -92,14 +141,31 @@ Edit Areas\Identity\Pages\Shared\_LoginPartial.cshtml in the project and change
 ### Preview 8
 1. Running xUnit tests fails to discover tests in with message `No test is available in <assembly> Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.`. To solve this update your `xunit.runner.visualstudio` package to version 2.4.3. This version is used in `dotnet new` templates, but existing projects targetting, or updating to net5.0 need to be updated manually.
 
-1. (Fixed in RC1, affects only Preview 8 and RC1 before 5.0.100-rc.1.20451.10) Running tests from net5.0 project referencing `Microsoft.NET.Test.SDK` older than 16.3.0 or older will fail to run tests with: `Microsoft.VisualStudio.TestPlatform.ObjectModel.SettingsException: Invalid settings 'RunConfiguration'. Invalid value 'net5.0' specified for 'TargetFrameworkVersion'.` To solve this update your `Microsoft.NET.Test.SDK` reference to `16.3.0` or newer, or update your SDK version to the latest RC1.
```

Comment:
Since this item is removed since it was fixed in RC1, does that mean the remaining "Known Issues" are still unfixed in the releases so far?